cabal-testsuite/PackageTests/Outdated/outdated_freeze.out

```diff
@@ -1,29 +1,54 @@
 # cabal v2-update
 Downloading the latest package list from test-local-repo
 # cabal outdated
+Configuration is affected by the following files:
+- cabal.project
+- cabal.project.freeze
 Outdated dependencies:
-base ==3.0.3.2 (latest: 4.0.0.0)
-template-haskell ==2.3.0.0 (latest: 2.4.0.0)
-binary ==0.8.5.0 (latest: 0.8.6.0)
+* project-config
+  - project config cabal.project.freeze
+    * binary ==0.8.5.0 (latest: 0.8.6.0)
+    * template-haskell ==2.3.0.0 (latest: 2.4.0.0)
+    * base ==3.0.3.2 (latest: 4.0.0.0)
```

Comment:
Could maybe these packages be ordered by name alphabetically?

The old order (base, t-h, binary) was reversed here (binary, t-h, base).
What governs the order in general?  Is it the order as in the freeze file?

Reply:
They are ordered now alphabetically, before it was the order in the cabal file or the freeze file.